Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationOverview.cshtml

```diff
@@ -133,44 +135,44 @@
                 @if (organisationDetails.IsTenderer())
                 {
                     <h3 class="govuk-heading-s govuk-!-margin-bottom-3">
-                        <a class="govuk-link" href="/organisation/@RouteData.Values["id"]/supplier-information">Complete supplier information</a>
+                        <a class="govuk-link" href="/organisation/@RouteData.Values["id"]/supplier-information">@StaticTextResource.Organisation_CompleteSupplierInformation</a>
                     </h3>
-                    <p class="govuk-body">Complete and share your supplier information to bid for procurements.</p>
+                    <p class="govuk-body">@StaticTextResource.Organisation_CompleteAndShareSupplierInformation</p>
                     <div class="govuk-section-break govuk-section-break--visible govuk-section-break--m"></div>
                 }
 
                 @if (organisationDetails.IsBuyer())
                 {
                     <h3 class="govuk-heading-s govuk-!-margin-bottom-3">
-                        <a class="govuk-link" href="/organisation/@RouteData.Values["id"]/manage-api-keys">@(isViewer ? "View" : "Manage") API keys@(organisationDetails.IsTenderer() ? " (for Buyers)" : "")</a>
+                        <a class="govuk-link" href="/organisation/@RouteData.Values["id"]/manage-api-keys">@(isViewer ? StaticTextResource.Organisation_ViewApiKeys : StaticTextResource.Organisation_ManageApiKeys) API keys@(organisationDetails.IsTenderer() ? StaticTextResource.Organisation_ApiKeysForBuyers : "")</a>
```

Comment:
The words "API keys" are not translated here. I'd expect them to be localized alongside the word view / manage:

```
<data name="Organisation_ViewApiKeys" xml:space="preserve">
    <value>View API keys</value>
  </data>
  <data name="Organisation_ManageApiKeys" xml:space="preserve">
    <value>Manage API keys</value>
  </data>
```

Reply:
Done